src/main/java/no/nav/familie/integrasjoner/personopplysning/PersonopplysningerController.kt

```diff
@@ -78,4 +89,13 @@ class PersonopplysningerController(private val personopplysningerService: Person
                                   defaultValue = "false") medHistorikk: Boolean): Ressurs<FinnPersonidenterResponse> {
         return success(personopplysningerService.hentIdenter(ident.ident, tema, medHistorikk))
     }
+
+
+    @PostMapping(produces = [MediaType.APPLICATION_JSON_VALUE], path = ["strengeste-adressebeskyttelse/"])
+    fun hentStrengesteAdressebeskyttelseForPersonMedRelasjoner(@RequestBody personIdent: PersonIdent,
+                                                               @RequestHeader(name = "Nav-Tema")
+                                                               tema: Tema): ADRESSEBESKYTTELSEGRADERING {
```

Comment:
Burde response wrappes i `Ressurs<ADRESSEBESKYTTELSEGRADERING>` ?

Reply:
Vi lurte litt på det samme, men valgte å ikke pakke inn i Ressurs. Hvis vi har noen gode argumenter for å pakke inn i ressurs endrer vi selvsagt :)

Reply:
Arrester meg gjerne, men er ikke alle responser fra `familie-integrasjoner` en `Ressurs` ? Isåfall tenker jeg det er fint å følge det mønsteret og returnere ressurs her også.

Reply:
Vi fulgte mønster - no.nav.familie.integrasjoner.tilgangskontroll.TilgangskontrollController. Returnerer ikke ressurser.

Reply:
Men det er et gyldig argument - vi bytter til ressurs :)

Reply:
Hehe ser den 👍